openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json

```diff
@@ -489,92 +490,92 @@
-    "whats-new": "What's new",
+    "whats-new": "What's New",
     "yes": "Yes"
   },
   "message": {
-    "access-to-collaborate": "Allow open access for anyone to join the team, view data, and collaborate",
+    "access-to-collaborate": "Allow open access for anyone to join the team, view data, and collaborate.",
     "active-users": "Display the number of active users.",
     "add-kpi-message": "Identify the Key Performance Indicators (KPI) that best reflect the health of your data assets. Review your data assets based on Description, Ownership, and Tier. Define your target metrics in absolute or percentage to track your progress. Finally, set a start and end date to achieve your data goals.",
-    "add-policy-message": "Policies are assigned to teams. In openMetadata, a policy is a collection of rules, which define access based on certain conditions. We support rich SpEL (Spring Expression Language) based conditions. All the operations supported by an entity are published. Use these fine grained operations to define the conditional rules for each policy. Create well-defined policies based on conditional rules to build rich access control roles.",
+    "add-policy-message": "Policies are assigned to teams. In OpenMetadata, a policy is a collection of rules, which define access based on certain conditions. We support rich SpEL (Spring Expression Language) based conditions. All the operations supported by an entity are published. Use these fine grained operations to define the conditional rules for each policy. Create well-defined policies based on conditional rules to build rich access control roles.",
     "add-role-message": "Roles are assigned to Users. In OpenMetadata, Roles are a collection of Policies. Each Role must have at least one policy attached to it. A Role supports multiple policies with a one to many relationship. Ensure that the necessary policies are created before creating a new role. Build rich access control roles with well-defined policies based on conditional rules.",
     "adding-new-tag": "Adding new tag on {{categoryName}}",
-    "admin-only-action": "Only admin can perform this action.",
+    "admin-only-action": "Only an Admin can perform this action.",
     "advanced-search-message": "Discover the right data assets using the syntax editor with and/or conditions.",
     "alerts-description": "Stay current with timely alerts using webhooks.",
-    "alerts-destination-description": "Send notifications to Slack, MS Teams, Email, or and use Webhooks.",
+    "alerts-destination-description": "Send notifications to Slack, MS Teams, Email, or use Webhooks.",
     "alerts-filter-description": "Specify the change events to narrow the scope of your alerts.",
     "alerts-trigger-description": "Trigger for all data assets or a specific entity.",
     "all-charts-are-mapped": "All charts are mapped with existing KPIs.",
     "and-followed-owned-by-name": "and followed team owned by {{userName}}",
```

Comment:
Would it be correct to say

The team you are following is owned by {{userName}}